docs/rules/computed-property-getters.md

```diff
@@ -0,0 +1,57 @@
+# Rule name: `computed-property-getters`
+
+## Enforce the consistent use of getters in computed properties
+
+Computed properties may be created with or without a `get` method. This rule ensures that the choice
+is consistent.
+
+## Options
+
+This rule takes a single string option
+
+String option:
+
+* `"never"` (default) getters are *never allowed* in computed properties
```

Comment:
I think the default should be to only allow them if there is also a setter present. don't ask me how to call that option... 😅

Reply:
As I started working on the docs for a change to add `always-with-setter` as an option here it strikes me as a strange thing to lint against.  A property created like this would throw an error when accessed anyway since there is no way to get it's value.

```
Ember.Object.extend({
    fullName: computed('firstName', 'lastName', {
        set() {
            //...
        }
    })
});
```

Am I confused about what your comment is asking for?

Reply:
what I mean is:

```js
// good
Ember.Object.extend({
    fullName: computed('firstName', 'lastName', function() {
        //...
    })
});

// bad
Ember.Object.extend({
    fullName: computed('firstName', 'lastName', {
        get() {
            //...
        }
    })
});

// good
Ember.Object.extend({
    fullName: computed('firstName', 'lastName', {
        get() {
            //...
        },
        set() {
            //...
        }
    })
});
```

Reply:
Thanks for the clarification @Turbo87, I've captured that now and updated the default.